single.php

```diff
@@ -17,7 +17,10 @@
 
 			get_template_part( 'template-parts/content', get_post_format() );
 
-			the_post_navigation();
+			the_post_navigation( array(
+				'prev_text' => '<span class="screen-reader-text">' . __( 'Previous', '_s' ) . '</span><span aria-hidden="true" class="nav-subtitle">' . __( 'Previous', '_s' ) . '</span> <span class="nav-title">%title</span>',
```

Comment:
Doesn't this just add the word "Previous/Next" twice? Once just for screen readers, and once visually with the `.nav-subtitle` class but hidden from screen readers? This could be simplified to just one `<span>` I think.